km/scripts/km_startup.py

```diff
@@ -135,14 +135,18 @@ def save_to_path(path, file):
 
     #  will not try a faucet if we're in mainnet or testnet
     if env in ['COMPOSE', 'K8S']:
-        try:
-            get_initial_coins(eth_address, 'ETH', config)
-        except RuntimeError as e:
-            logger.critical(f'Failed to get enough ETH or ENG to start - {e}')
-            sys.exit(-2)
-        except ConnectionError as e:
-            logger.critical(f'Failed to connect to remote address: {e}')
-            sys.exit(-1)
+        while True:
+            try:
+                get_initial_coins(eth_address, 'ETH', config)
+                break
+            except RuntimeError as e:
+                logger.critical(f'Failed to get enough ETH or ENG to start - {e}')
+                sys.exit(-2)
+            except ConnectionError as e:
+                logger.critical(f'Failed to connect to remote address: {e}')
+                sleep_time = 30
```

Comment:
Not a fan of this. Try one of the others:
- Use a SLEEP_TIME global
- stick 30 in the sleep since it's obvious what you're trying to say anyway
- Add it as a configuration parameter